[2024-06-24T12:04:05.319+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T12:04:05.336+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: httpsensor.process_data manual__2024-06-24T12:04:03.440398+00:00 [queued]>
[2024-06-24T12:04:05.341+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: httpsensor.process_data manual__2024-06-24T12:04:03.440398+00:00 [queued]>
[2024-06-24T12:04:05.341+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-24T12:04:05.349+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): process_data> on 2024-06-24 12:04:03.440398+00:00
[2024-06-24T12:04:05.352+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=493) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T12:04:05.353+0000] {standard_task_runner.py:63} INFO - Started process 494 to run task
[2024-06-24T12:04:05.353+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'httpsensor', 'process_data', 'manual__2024-06-24T12:04:03.440398+00:00', '--job-id', '440', '--raw', '--subdir', 'DAGS_FOLDER/sensors.py', '--cfg-path', '/tmp/tmp9k4alt_f']
[2024-06-24T12:04:05.354+0000] {standard_task_runner.py:91} INFO - Job 440: Subtask process_data
[2024-06-24T12:04:05.380+0000] {task_command.py:426} INFO - Running <TaskInstance: httpsensor.process_data manual__2024-06-24T12:04:03.440398+00:00 [running]> on host e3e60e62d12c
[2024-06-24T12:04:05.426+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='httpsensor' AIRFLOW_CTX_TASK_ID='process_data' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T12:04:03.440398+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-24T12:04:03.440398+00:00'
[2024-06-24T12:04:05.427+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T12:04:05.935+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T12:04:05.936+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/sensors.py", line 19, in query_api
    print(response.txt)
          ^^^^^^^^^^^^
AttributeError: 'Response' object has no attribute 'txt'. Did you mean: 'text'?
[2024-06-24T12:04:05.955+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=httpsensor, task_id=process_data, run_id=manual__2024-06-24T12:04:03.440398+00:00, execution_date=20240624T120403, start_date=20240624T120405, end_date=20240624T120405
[2024-06-24T12:04:05.981+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 440 for task process_data ('Response' object has no attribute 'txt'; 494)
[2024-06-24T12:04:06.007+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-24T12:04:06.017+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-24T12:04:06.019+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
